src/screen/dashboard.rs

```diff
@@ -777,6 +777,9 @@ impl Dashboard {
                                     (window::toggle_fullscreen(), None)
                                 }
                             },
+                            command_bar::Command::Application(application) => match application {
+                                command_bar::Application::Quit => (iced::exit(), None),
+                            },
```

Comment:
We need to use `Dashboard::exit` which runs our graceful exit logic. This is called during `CloseRequested` in `main`, such as:

```rs
                        window::Event::CloseRequested => {
                            if let Screen::Dashboard(dashboard) = &mut self.screen {
                                return dashboard.exit().map(Message::Dashboard);
                            } else {
                                return iced::exit();
                            }
                        }
```